tests/ui/consts/miri_unleashed/mutable_references.rs

```diff
@@ -30,9 +30,11 @@ const BLUNT: &mut i32 = &mut 42;
 //~^ ERROR: it is undefined behavior to use this value
 //~| pointing to read-only memory
 
-const SUBTLE: &mut i32 = unsafe { static mut STATIC: i32 = 0; &mut STATIC };
-//~^ ERROR: it is undefined behavior to use this value
-//~| static
```

Comment:
Please keep this annotation. It ensures that the error message mentions the right term, to make sure that the const fails to evaluate for the right reason.

(If you removed similar annotations anywhere else, please also restore them.)

Reply:
I have checked the diffs and this is the only annotation in concern which is restored now.

In addition, the annotation has been updated after rebasing onto the latest `master`.

Reply:
Looks good, thanks!